### ISSUE TITLE
Example of how to implement Upload-Length-Deferred

### ISSUE THREAD

**Issue by khabubuphathu** [tusdotnet/tusdotnet.wikisource#5]
I have been trying to use tusdotnet with tus-client0js to upload chunks of a user's display media using Upload-Length-Deferred, but I keep getting an error that 
```
tus: unexpected response while uploading chunk, originated from request (method: PATCH, url: /files/:fileId, response code: 400, response text: Header Upload-Length must be specified as this file was created using Upload-Defer-Length, request id: n/a).
```

Is there additional configuration needed in tusdotnet needed to make this work?

**Comment by smatsson** [comment]
Hi,

Unfortunately this is a bug in tusdotnet related to ambiguous language in the tus spec. The spec was recently updated to clarify this part but tusdotnet has not yet been updated with the new spec (see https://github.com/tus/tus-resumable-upload-protocol/pull/181).

I have created an issue in the main repo for this problem and will address it as soon as I'm back from vacation. https://github.com/tusdotnet/tusdotnet/issues/182

**Comment by smatsson** [comment]
The bug has been fixed in 2.6.2 which is now on nuget. https://www.nuget.org/packages/tusdotnet/2.6.2
